Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -4139,6 +4143,19 @@ public struct UnmanagedMemory : IDisposable
 
             public ActionMapIndices* mapIndices;
 
+            // This is Anthony's version of block allocation from a blob
+            private byte* AllocFromBlob(ref byte* top, int size)
+            {
+                if (size == 0)
+                    return null;
+                
+                var allocation = top;
+
```

Comment:
Minor: I think we can remove these blank lines in-between?

Reply:
Applied changes according to all suggestions in this function.